src/olympia/activity/management/commands/repudiate_token.py

```diff
@@ -0,0 +1,36 @@
+import logging
+from optparse import make_option
+
+from django.core.management.base import BaseCommand, CommandError
+
+from olympia.activity.models import ActivityLogToken
+
+log = logging.getLogger('z.amo.activity')
+
+
+class Command(BaseCommand):
+    args = u'<token_uuid token_uuid ...>'
+    help = u'Force expire a list of Activity Email tokens.'
+
+    option_list = BaseCommand.option_list + (
+        make_option('--version_id', action='store', type='long',
+                    dest='version_id',
+                    help='Expire all tokens on this version.'),
+    )
+
+    def handle(self, *args, **options):
+        version_pk = options.get('version_id')
+        if len(args) > 0:
+            done = [t.expire() for t in ActivityLogToken.objects.filter(
+                uuid__in=args)]
+            log.info(u'%s tokens (%s) expired' % (len(done), ','.join(args)))
+            if version_pk:
+                print 'Warning: --version_id ignored as tokens provided too'
+        elif version_pk:
+            done = [t.expire() for t in ActivityLogToken.objects.filter(
+                version__pk=version_pk)]
+            log.info(
+                u'%s tokens for version %s expired' % (len(done), version_pk))
+        else:
```

Comment:
Added in https://github.com/mozilla/addons-server/pull/3564/commits/fbc75bc6f61d940ee49716018da31c91d82ce8af